WordPress/Classes/ViewRelated/System/FancyAlerts+AztecAnnouncement.swift

```diff
@@ -51,11 +51,10 @@ extension FancyAlertViewController {
         }
 
         let addConfetti: (FancyAlertButtonHandler) = { controller in
-            guard let imageView = controller.headerImageView else { return }
+            guard let imageView = controller.headerImageView, let imageViewHolder = imageView.superview else { return }
 
             let confettiView = ConfettiView.aztecAnnouncementConfettiView()
-            confettiView.frame = imageView.bounds.insetBy(dx: Constants.confettiViewInset,
-                                                          dy: Constants.confettiViewInset)
+            confettiView.frame = imageViewHolder.bounds.offsetBy(dx: 0, dy: Constants.confettiViewOffset)
```

Comment:
Can you take another look at this? I think the confetti view frame ends up being quite different to on `develop`.